### PR TITLE
Add dictionaries and Update 'DeduplicateAndSortDictionaries'

### PR DESCRIPTION
**Pull request by MechanicKim**
Thanks @Ka1wa

I tested it with maven and the result was success.
I updated `DeduplicateAndSortDictionaries.scala`.

@nlpenguin Please review my updated code.

### CODE REVIEW

**Comment by nlpenguin** [comment]
Thank you for the PR @MechanicKim! The dictionary files won't be loaded until you fix `src/main/scala/org/openkoreantext/processor/util/KoreanDictionaryProvider.scala`. Please add the files in this file.

File: src/main/scala/org/openkoreantext/processor/tools/DeduplicateAndSortDictionaries.scala [hunk 1]
```diff
@@ -35,39 +36,39 @@ object DeduplicateAndSortDictionaries extends Runnable  {
...
-    "auxiliary/determiner.txt", "auxiliary/exclamation.txt", "auxiliary/conjunctions.txt",
-
-    "josa/josa.txt", "typos/typos.txt",
-
-    "verb/eomi.txt", "verb/pre_eomi.txt", "verb/verb.txt", "verb/verb_prefix.txt"
+  private val EXCEPTION_RESOURCES = Seq(
+    "example_chunks.txt", "example_tweets.txt"
   )
+  
+  def work(file: File) {
```
Comment (nlpenguin) [inline src/main/scala/org/openkoreantext/processor/tools/DeduplicateAndSortDictionaries.scala]: rename to `perform`.

**Comment by Ka1wa** [comment]
Should probably also rename the folder to mecab-ko-dic and include the copyright statement and one of the licenses to comply with them.

**Comment by MechanicKim** [comment]
@Ka1wa  I'm sorry I'm late answer.
I added new dictionary and ran JUnit, but the test is failed.
I found that a word '**착한**' is in **NNG.txt**.
I know that the word is adjective, so I can't merge this PR.
Could you please review your dictionary?

**Comment by Ka1wa** [comment]
Well it's not my dictionary, I'd just figured with a dictionary of mecab-ko's size it would benefit. Is there any way that I can run these tests and remove fault words myself for this PR?

**Pull request closed without merging**